src/transformers/onnx/config.py

```diff
@@ -368,7 +370,12 @@ def generate_dummy_inputs(
                     tokenized_input[k] = [v[i : i + num_choices] for i in range(0, len(v), num_choices)]
                 return dict(tokenized_input.convert_to_tensors(tensor_type=framework))
             return dict(preprocessor(dummy_input, return_tensors=framework))
-        elif isinstance(preprocessor, FeatureExtractionMixin) and preprocessor.model_input_names[0] == "pixel_values":
+        elif isinstance(preprocessor, ImageProcessingMixin):
+            if preprocessor.model_input_names[0] != "pixel_values":
+                raise ValueError(
+                    f"The `preprocessor` is an image processor ({preprocessor.__class__.__name__}) and expects"
+                    f' `model_input_names[0]` to be "pixel_values", but got {preprocessor.model_input_names[0]}'
+                )
```

Comment:
I think it's better to throw an error if `ImageProcessingMixin` is not with `pixel_values`.

Reply:
I removed `FeatureExtractionMixin`, but `OwlViT` still uses `OwlViTFeatureExtractor` (and there is no `OwlViTImageProcessor`).

So with this change, ONNX test for OwlViT will fail

Reply:
Agreed